pootle/apps/pootle_store/fields.py

```diff
@@ -187,6 +187,7 @@ def _update_store_cache(self):
                 from pootle_store.filetypes import factory_classes
 
                 store_obj = factory.getobject(self.path,
+                                              localfiletype=self.instance.translation_project.project.localfiletype,
```

Comment:
Hi,
Please see https://github.com/translate/translate/pull/3270
The issue is that the filetype is resolved from the extension. The issue is that some filetype, for example gwt properties files (java properties files + plurals handling) use the same extension that java properties files. So pootle use the wrong format